fastlane/metadata/en-US/app_store_screenshot_1.txt

```diff
@@ -0,0 +1,2 @@
+A simple note taking
+experience
```

Comment:
These files keep disappear every time I run the lane to fetch localizations from GlotPress.

I remember having to put them here to get the screenshots composition to work back when I first worked on it, but I was such a noob then 😅 I'll see if I can move them in a place where they won't disappear as a followup.